### PR TITLE
Add october programs and past programs

### LINKED ISSUE #70: Front-end: October Events
# Purpose
OSL is hosting some October events that we need to put on the site ASAP! Please list the new events under an overarching "Upcoming Programs" section, and put the June events under a new section called "Past Programs."

# Description
Update the Programs Page with the events in this [doc ](https://docs.google.com/document/d/1ifcnDE_C0cWaWv0amwUXkKW4MDRafQXD/edit?usp=sharing&ouid=105775914785337038980&rtpof=true&sd=true)
- [x] New Upcoming Programs section
- [x] Add October Events
- [x] Migrate June Events to a Past Programs section

Here is a very blurry screenshot the Davids gave us haha
Screen Shot 2022-10-08 at 9 35 23 PM

# Acceptance Criteria
- [ ] Please attach before and after screenshots of the changes you've made to these pages.

### PR DESCRIPTION
**Pull request by al-chang**
### ℹ️ Issue

Closes #70 

### 📝 Description

- Add October events 
- Add past events section

### ✔️ Verification

Before:

image

After:

image

### CODE REVIEW

File: lib/components/programs-list/index.tsx [hunk 1]
```diff
@@ -73,7 +34,9 @@ const ProgramListing: React.FC<ProgramListingProps> = ({
       />
       <Flex flexDirection="column" alignItems="start">
         <Heading>{title}</Heading>
-        <Text fontSize="lg">{description}</Text>
+        <Text fontSize="lg" whiteSpace="pre-line">
```
Comment (thejamielin) [inline lib/components/programs-list/index.tsx:37]: Why are we adding the `whiteSpace="pre-line"` attribute? This replaces white space with more than 2 spaces with a single space. Is our content just not formatted correctly?

**Review by thejamielin — CHANGES_REQUESTED** [review]
So far so good! Are we able to get the header divider in the Programs page span the whole page (rather than just fit under "October 2022 Online Events")?

File: pages/programs/index.tsx [hunk 2]
```diff
@@ -1,23 +1,123 @@
...
+        </div>
+        <ProgramsList programListings={programs} />
+        <div data-cy="past-programs-heading">
+          <OSLHeader
+            header="Past Online Events"
+            style={{ marginTop: '60px', marginBottom: '30px' }}
+          />
         </div>
-        <ProgramsList />
+        <ProgramsList programListings={pastPrograms} />
```
Comment (thejamielin) [inline pages/programs/index.tsx:120]: I really like how you pulled this out of the programs-list component!

File: pages/programs/index.tsx [hunk 3]
```diff
@@ -1,23 +1,123 @@
...
+  {
+    title: 'MONTHLY WRITING GROUP',
+    description:
+      "Ready to focus on your writing? Join OutstandingLife's new monthly writing group. \
+    In this monthly writing workshop, we will discuss a short passage or \
+    poem that effectively uses a particular form or point of view. After \
+    considering how the technique enhances the piece, we will use some \
+    aspect of the text as a prompt to jumpstart our own writing. The \
+    focus will be on short, in-class writing and discussion. Suitable for \
+    writers with some experience. \n\n Bios of workshop leaders \n\n \
```
Comment (thejamielin) [inline pages/programs/index.tsx:15]: For some reason, in Chrome and Microsoft Edge, I see
image

which doesn't have bios of workshop leaders on new lines. Is this something that you guys encountered at all or is my computer just buggy?

**Review by thejamielin — COMMENTED** [review]
I like the changes you guys made to make the code more readable (taking the ProgramsList and making it a props of Programs)! Made a couple more comments

File: pages/programs/index.tsx (hunk 3 above)
Comment (adriannaliu) [reply pages/programs/index.tsx:15]: I don't have time to check this out right now but +1 to Jamie's comment here

**Review by adriannaliu — COMMENTED** [review]
Looks good so far guys! +1 to all of Jamie's comments

I'll also send you guys the link to the design files soon, but the gist is this: would it be possible to have a dropdown menu for the Programs tab, and have that take you to either Past Events or Upcoming Events as separate pages?

File: pages/programs/index.tsx [hunk 4]
```diff
@@ -1,23 +1,123 @@
-import { ProgramsList } from '../../lib/components/programs-list';
-import { Text, Container, Flex } from '@chakra-ui/react';
+import { ProgramListingProps, ProgramsList } from '../../lib/components/programs-list';
+import { Container, Flex } from '@chakra-ui/react';
 import OSLHeader from '../../lib/components/header';
 
+const programs: ProgramListingProps[] = [
+  {
+    title: 'MONTHLY WRITING GROUP',
+    description:
```
Comment (rymaju) [inline pages/programs/index.tsx:8]: Is this really the best way to store this information? At this point a line by line string is probably a bad representation.

File: pages/programs/index.tsx [hunk 5]
```diff
@@ -1,23 +1,123 @@
...
+      "Amy Hoffman will be reading from her new novel, Dot and Ralfie. Dot & Ralfie is about a lesbian couple facing the \
+physical, emotional, and relationship challenges \
+of aging. \
+\n \
+Co-sponsored by OutstandingLife and 2Life Communities. https://2lifecommunities.org/ \
+\n \
+Copies of Dot and Ralfie and available at Paper Cuts Bookshop in Jamaica Plain. https://papercutsjp.com \
+\n \
+\n \
```
Comment (rymaju) [inline pages/programs/index.tsx:53]: Did you type this all in manually?

**Review by rymaju — COMMENTED** [review]
We should consider better ways of storing free form text

**Review by thejamielin — APPROVED** [review]
we should go back to these issues, but we need to merge this by today!

**Review by adriannaliu — APPROVED** [review]

**Pull request merged**